network/src/validator_network/network_builder.rs

```diff
@@ -302,18 +261,19 @@ impl NetworkBuilder {
             &counters::PENDING_CONNECTIVITY_MANAGER_REQUESTS,
         );
         self.conn_mgr_reqs_tx = Some(conn_mgr_reqs_tx);
-        let peer_id = self.peer_id;
         let trusted_peers = self.trusted_peers.clone();
         let seed_peers = self.seed_peers.clone();
         let max_connection_delay_ms = self.max_connection_delay_ms;
-        let connectivity_check_interval_ms = self.connectivity_check_interval_ms;
         let pm_conn_mgr_notifs_rx = self.add_connection_event_listener();
         let conn_mgr = self.executor.enter(|| {
             ConnectivityManager::new(
-                peer_id,
+                self.peer_id,
                 trusted_peers,
                 seed_peers,
-                interval(Duration::from_millis(connectivity_check_interval_ms)).fuse(),
+                interval(Duration::from_millis(
+                    self.config.connectivity_check_interval_ms,
+                ))
+                .fuse(),
```

Comment:
either always use `self.___` below or never.  The middle ground where `self` is used sometimes but not always is  between two worlds.

Reply:
Sounds good, I'll update those as well